Statistics/Histograms/Histogram.cs

```diff
@@ -182,7 +182,7 @@ public void AddObservationToHistogram(IData data)
                 }
                 newBinCounts[BinCounts.Length + quantityAdditionalBins-1] += 1;
                 BinCounts = newBinCounts;
-                double newMax = (BinCounts.Length +1) * BinWidth;
+                double newMax = Min + BinCounts.Length * BinWidth; //is this right?
```

Comment:
i think you need a parenthesis (BinCounts.Length * BinWidth) - it is easier to read.